cardano-node/app/cardano-node.hs

```diff
@@ -85,7 +85,7 @@ parseVersionCmd =
 runVersionCommand :: IO ()
 runVersionCommand =
     putTextLn $ mconcat
-      [ "cardano-cli ", renderVersion version
+      [ "cardano-node ", renderVersion version
```

Comment:
I thought a fix for that had already been merged.

Reply:
Specifically https://github.com/input-output-hk/cardano-node/pull/1263/commits/46142ef428cf66d55f51dce6eac2f6966a193e0c

Reply:
So it has - rebased.